fhir-model/src/main/java/com/ibm/fhir/model/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -904,14 +904,14 @@ private IllegalArgumentException unexpectedNumberOfArguments(int arity, String f
 
             switch (operator) {
             case "is":
-                if (!isSingleton(nodes)) {
+                if (nodes.size() > 1) {
```

Comment:
the description in the exception messages seems doesn't align with the codes... 
so, only 0 and 1 are allowed or really only 1 is allowed?

Reply:
its a bit fuzzy.  the spec says only 1 is allowed, but in practice we need to allow 0.  i think its ok that the error message says only 1 is allowed because now its only shown in cases where the size > 1.

Reply:
see https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=25189 for more info